### PR TITLE
UGENE-7238. Step 3: build MSA editor 'Align' menu using new framework…

### PR DESCRIPTION
**Pull request by yalgaer**
This is the next step of adding custom menu to "Align sequence to alignment" action.

This step demonstrates how the framework introduced in the Step 2 (https://github.com/ugeneunipro/ugene/pull/384) will be used and correctly rewrites "Align" button menu.

In the next step I will add a popup menu to "Align sequence to alignment"  button.

No changes is expected in UGENE UX as the result of this PR: internal refactoring only.

### CODE REVIEW

File: src/corelibs/U2View/src/ov_msa/MSAEditor.h [hunk 1]
```diff
@@ -150,31 +150,38 @@ protected slots:
...
+    QAction *convertRnaToDnaAction = nullptr;
 
 private:
-    PairwiseAlignmentWidgetsSettings *pairwiseAlignmentWidgetsSettings;
+    PairwiseAlignmentWidgetsSettings *pairwiseAlignmentWidgetsSettings = nullptr;
     MSAEditorTreeManager treeManager;
 };
 
+/** Set of custom menu actions in MSA editor. */
+class U2VIEW_EXPORT MsaEditorMenuType {
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MSAEditor.h]: Inherit it from `GObjectViewMenuType` to show the fact they are connected

File: src/corelibs/U2View/src/ov_msa/MSAEditorConsensusArea.cpp [hunk 2]
```diff
@@ -48,8 +48,10 @@ QString MSAEditorConsensusArea::getConsensusPercentTip(int pos, int minReportPer
     return MSAConsensusUtils::getConsensusPercentTip(editor->getMaObject()->getMultipleAlignment(), pos, minReportPercent, maxReportChars);
 }
 
-void MSAEditorConsensusArea::sl_buildMenu(GObjectView * /*view*/, QMenu *menu, const QString &) {
-    buildMenu(menu);
+void MSAEditorConsensusArea::sl_buildMenu(GObjectView * /*view*/, QMenu *menu, const QString &menuType) {
+    if (menuType == GObjectViewMenuType::CONTEXT || menuType == GObjectViewMenuType::STATIC) {
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MSAEditorConsensusArea.cpp]: After you inherited, it's better to use `MsaEditorMenuType::CONTEXT`, I suppose

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]
Minor comments

File: src/corelibs/U2View/src/ov_msa/MSAEditor.h (hunk 1 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_msa/MSAEditor.h]: Fixed

File: src/corelibs/U2View/src/ov_msa/MSAEditorConsensusArea.cpp (hunk 2 above)
Comment (yalgaer) [reply src/corelibs/U2View/src/ov_msa/MSAEditorConsensusArea.cpp]: Fixed

**Pull request merged**